tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java

```diff
@@ -96,7 +97,8 @@ public static IStatus importGradleProject(File dir, IProgressMonitor monitor) th
 
 		build.synchronize(NewProjectHandler.IMPORT_AND_MERGE, AsyncHandler.NO_OP);
 
-		waitImport();
+		// We should import asynchronous or not wait the gradle job finished 
```

Comment:
This change should be another PR not dealing with Watch.  Can you create another ticket for this issue?